tests/TestCase.php

```diff
@@ -13,54 +11,37 @@ public function setUp()
     {
         parent::setUp();
 
-        $this->setUpDatabase($this->app);
+        $this->loadMigrationsFrom(realpath(__DIR__.'/fixtures/migrations'));
     }
 
     protected function getPackageProviders($app)
     {
-        return [TranslatableServiceProvider::class];
+        return [
+            \Orchestra\Database\ConsoleServiceProvider::class,
```

Comment:
Is this needed? Seems to me that tests run perfectly without using this service provider.